C/common/include/asset_tracking.h

```diff
@@ -21,6 +21,8 @@
 #include <condition_variable>
 #include <storage_client.h>
 
+#define MIN_ASSET_TRACKER_UPDATE	500 // The minimum interval for asset tracker updates
```

Comment:
Can even 500 ms setting be considered too aggressive in some cases given that these changes go to (separate) foglamp.db and is not competing with ingestion/reading of readings?

Reply:
This is here for compatibility reasons as the current value is 500ms

Reply:
Ok, got it. This value may be reduced even below 500 ms. 

Can you say what was the original issue that required this change?

Reply:
It was a request from RTE as they saw the audit logger taking a lot of bandwidth from the storage service